tests/unit_tests/lcls_tools/common/measurements/test_screen_beam_profile.py

```diff
@@ -0,0 +1,119 @@
+from lcls_tools.common.measurements.screen_beam_profile_measurement import (
+    ScreenBeamProfileMeasurement,
+)
+from lcls_tools.common.data.fit.projection import ProjectionFit
+from lcls_tools.common.data.fit.methods import GaussianModel
+from lcls_tools.common.image.processing import ImageProcessor
+from lcls_tools.common.image.roi import ROI
+from lcls_tools.common.devices.device import Metadata
+from lcls_tools.common.devices.screen import (
+    ScreenControlInformation,
+    ScreenPVSet,
+    Screen,
+)
+import numpy as np
+import unittest
+
+
+class ScreenTest(Screen):
+    @property
+    def image(self) -> np.ndarray:
+        return self._image
+
+    @image.setter
+    def image(self, image):
+        self._image = image
+
+
+class TestScreenBeamProfileMeasurement(unittest.TestCase):
+    def __init__(self, methodName: str = "runTest") -> None:
+        super().__init__(methodName)
+        self.test_instantiate_pydantic_objects()
+
+    def create_test_image(self, size: tuple, center: list, radius: int):
+        # make img that is a circle in the center of the image with known
```

Comment:
I would prefer if we use a test file. That way functionality changes would not impact us. We would know we are always testing against the same image.